recipes/attune/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "attune" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "071e2ff296681cab3194da35646bff7b864ab981e5b650ea3d5be5379615869f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - setuptools
+    - pytest-runner
+  run:
+    - python >=3.5
+    - WrightTools >= 3.2.2
+    - numpy
```

Comment:
There should not be a space after the equals sogn

Reply:
:+1: